tests/variable-axis.sil

```diff
@@ -0,0 +1,7 @@
+\begin[papersize=a6]{document}
+\neverindent
+\nofolios
+\font[family=Tourney,size=48pt]
+\font[variations="wght=150,wdth=120"]{Foo bar}\par
+\font[variations="wght=750,wdth=80"]{Foo bar}
```

Comment:
This test is working for me as written here, but I was a bit surprised when working this up to find this did *not* work at all:

```suggestion
\font[weight=750,variations=wdth=80]{Foo bar}
```

When including *any* variations parameter value at all, the weight option gets ignored. It works fine to select any value from the weight axis right up until any other axis is specified, then it stops working an one must add the `wght` axis to the variations to get the effect back.

Is there some way we can map the axis that correspond to parameters we already have? I have to imagine this is going to affect more combinations too, such as when `style=Italic` is part of a slant axis.

Reply:
The `weight` option should be respected if the font manager finds a named instance matching it (the same for other style settings we ask for in the font manager). If this is not working, then something is broken.

But generally, there might nit be a named instance matching the style and it will be ignored. What can we do it to treat `weigh`, `width` (or `stretch` if SILE supports any) and `style=italic`/`style=oblique` the same way we treat font size; set the corresponding variation axes to their values.

So, what is the list of styles we want to apply as variations?